web/src/js/components/Header/FlowMenu.tsx

```diff
@@ -29,9 +29,6 @@ export default function FlowMenu(): JSX.Element {
     const selectedFlows = useAppSelector((state) => state.flows.selected);
     const flow = selectedFlows[0];
 
-    const hasSingleFlowSelected = useAppSelector(
-        (state) => state.flows.selected.length === 1,
-    );
     const canResumeOrKillAny = selectedFlows.some(canResumeOrKill);
```

Comment:
Needs something like this as well I believe

Reply:
what do you mean?
I am disabling the button with the following: `disabled={!selectedFlows.some(canReplay)}`.

Reply:
Can't read, thanks. :)